scripts/kind-e2e/e2e.sh

```diff
@@ -183,39 +199,36 @@ function test_connection() {
 
 function update_subm_pods() {
     echo Removing submariner engine pods...
-    kubectl delete pods -n submariner -l app=submariner-engine
-    kubectl wait --for=condition=Ready pods -l app=submariner-engine -n submariner --timeout=60s
+    kubectl --cluster=$1 delete pods -n submariner -l app=submariner-engine
```

Comment:
--cluster or ---context?

Reply:
@skitt did you mean cluster or context? I'm confused,

Reply:
@mangelajo context; see #109 for the update. I guess I need to stop chaining PRs, GitHub makes them too complicated to review :-(.